server/headless-agent/package-lock.json

```diff
@@ -848,7 +848,7 @@
         "@fluidframework/gitresources": "^0.1012.2",
         "@fluidframework/protocol-definitions": "^0.1012.2",
         "@fluidframework/server-services-client": "^0.1012.2",
-        "@types/nconf": "^0.0.37",
+        "@types/nconf": "^0.10.0",
```

Comment:
Looks like the pipeline is failing here. The server packages in headless-agent are pretty old. Might wanna just leave headless-agent alone?

Reply:
Headless-Agent was missing a devDependency on '@types/nconf'.

(Previously was working "by luck" as a result of Lerna hoisting and/or loose npm module resolution.)